guides/steam-setup/connect-with-steam.md

```
@@ -0,0 +1,58 @@
+﻿# Connect with Steam Using the Steam Transport
+If you're looking for a quick and easy guide that will teach you how to connect with players over steam then you've come to the right place. Before we begin let's clarify what this document is and is not below:
+#### This is:
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)